src/node/progress_tracker.h

```diff
@@ -635,18 +635,6 @@ namespace ccf
     void rollback(ccf::SeqNo rollback_seqno, ccf::View view)
     {
       std::unique_lock<SpinLock> guard(lock);
-      if (rollback_seqno < highest_commit_level)
```

Comment:
That seemed like a good sanity check, is it gone because it's not in fact correct for a reason that's not obvious to me now, or just because it's redundant with a check somewhere else?

Reply:
its wrong here, commit its a local property so only less than f+1 machines may have reached commit. The correct check is in raft.h (same as for CFT) where we do not rollback anything that is globally committed.